user-guide/Troubleshooting/Procedures/TroubleshootingSLScriptingOutOfMemoryException.md

```diff
@@ -0,0 +1,54 @@
+---
+uid: TroubleshootingSLScriptingOutOfMemoryException
+---
+
+# Investigating OutOfMemoryException occurrence
+
+> [!IMPORTANT]
+> This section might include some information that is only applicable to Skyline employees and/or links that are only accessible to Skyline employees.
+
+In case you experience an SLScripting crash because the process ran out of memory or you observe a memory leak to be present in the SLScripting process, analyzing a memory dump of the SLScripting process can provide more info on what is causing the leak.
+
+Note that by default a single instance of the SLScripting process is spun up by DataMiner. This process is responsible for executing the QActions of protocols. QActions execute C# code which is executed by the CLR of the .NET Framework. Therefore, if you observe a memory leak in SLScripting, a good stating point is to analyze the managed memory of the SLScripting process. Note however that in some situations a leak could be present in unmanaged memory, for example when unmanaged resources are not cleaned up correctly.
```

Comment:
Maybe move the note about unmanaged leak to a [!NOTE] banner at the bottom.
Something like "if you can't identify the objects that are consuming the memory, it is possible that the leak could be present in unmanaged memory. Review the usage of IDisposable objects or other code or libraries that could allocate unmanaged memory."

Reply:
added caution note